src/components/Outbox.vue

```diff
@@ -127,4 +132,14 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+.second-column {
```

Comment:
…licate code?

Reply:
no, im styling the full container list, and then the header separately

Reply:
You could create another "layout" component, since MailboxThread and Outbox have a very similar structure

Reply:
I discussed this with Marco and agreed to do it in a separate pr as a refactoring pr, because its some work and it doest add that much to this pr.